Analyzer/interface/Regions.h

```diff
@@ -388,15 +389,18 @@ void bckgEstimate(Region& b, Region& c, Region& bc, Region& a, Region& d, std::s
     delete RNG;
 }
 
-TH1F* bckgEstimate_fromHistos(TH2F* eta_cutIndex_regA, TH2F* eta_cutIndex_regB, TH3F* ih_eta_cutIndex_regB, TH3F* eta_p_cutIndex_regC, int cutIndex=3, bool mass_rebin=true, int nPE=100){
-    TH1F* eta_regA = (TH1F*)eta_cutIndex_regA->ProjectionY("_proj",cutIndex+1,cutIndex+1);
-    TH1F* eta_regB = (TH1F*)eta_cutIndex_regB->ProjectionY("_proj",cutIndex+1,cutIndex+1);
+void bckgEstimate_fromHistos(TH2F* mass_cutIndex, TH2F* eta_cutIndex_regA, TH2F* eta_cutIndex_regB, TH3F* ih_eta_cutIndex_regB, TH3F* eta_p_cutIndex_regC, TH1F* H_A, TH1F* H_B, TH1F* H_C, int cutIndex=3, int nPE=100){
+    TH1F* eta_regA = (TH1F*)eta_cutIndex_regA->ProjectionY("_projA",cutIndex+1,cutIndex+1);
+    TH1F* eta_regB = (TH1F*)eta_cutIndex_regB->ProjectionY("_projB",cutIndex+1,cutIndex+1);
     ih_eta_cutIndex_regB->GetXaxis()->SetRange(cutIndex+1,cutIndex+1);
-    TH2F* ih_eta_regB =  (TH2F*)ih_eta_cutIndex_regB->Project3D("zy");
+    TH2F* ih_eta_regB =  (TH2F*)ih_eta_cutIndex_regB->Project3D("zyB");
```

Comment:
@dapparu what is the `B` doing in `project3D("zyB")?

Reply:
ah it's just for the name of the projected histo? I didnt know that it's fine to just pass that in the `project3D`